eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -126,6 +126,7 @@ if ($relatedTypeSpecProjectFolder) {
           })
         } else {
             $relativeSdkPath = Resolve-Path $sdkProjectFolder -Relative
+            $specRepoRoot = $swaggerDir -replace "/specification", ""
```

Comment:
Are we sure all these paths have "/" in the front? Should we try to use path apis instead if all we are trying to do is remove the specification folder from the path.

Reply:
Yes, the slash must be `/` as the former code replaces `\` by `/`.

Reply:
OK I just want to be sure there aren't cases where it might not have a slash at all as this will fail in that case.